app/controllers/estimates_controller.rb

```diff
@@ -4,32 +4,34 @@ def new
   end
 
   def create
-    @model = CfeService.call(cfe_session_data(:estimate_id))
-    track_page_view(assessment_id: params[:estimate_id], page: :view_results)
-    render :show
+    redirect_to estimate_path(params[:estimate_id])
+  end
+
+  def show
+    @model = CfeService.call(session_data)
+    track_page_view(page: :view_results)
```

Comment:
Not sure I'm happy with this change. IMHO the 'GET' should not invoke the service to create the assessment . The 'create' call was a little strange in rendering the 'show' template - this could just have been called 'create' (c.f. rails generated templates, which do just that but common-up the 'created' and 'shown' views for DRY purposes.

Reply:
OK. I do think there's a case for turning this into a GET/show (we invoke the service on other GETs like `print` and `download`, and I don't think the result's page should fundamentally be less bookmarkable than any of the other pages. But a change like that is controversial enough that it doesn't belong in this PR.